apps/web/types/rate-limit.ts

```diff
@@ -0,0 +1,16 @@
+import z from "zod";
+
+const RateLimitResponseDataSchema = z.object({
+  target: z.string(),
```

Comment:
Let's narrow the type to avoid incorrect values

```suggestion
  target: z.enum(["API"]),
```